src/python_testing/TC_DeviceConformance.py

```diff
@@ -76,17 +76,23 @@ def record_warning(location, problem):
 
         success = True
         allow_provisional = self.user_params.get("allow_provisional", False)
+        # TODO: automate this once https://github.com/csa-data-model/projects/issues/454 is done.
+        provisional_cluster_ids = [0x050F, 0x0062]
```

Comment:
```suggestion
        provisional_cluster_ids = [
            0x050F, # Content Control 
            0x0062, # Scenes Management
        ]
```

Reply:
I added them from the objects file, since they're there.